profile/README.md

```diff
@@ -0,0 +1,64 @@
+[<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/git/git-original-wordmark.svg" alt="git" width="60" height="60"/>](https://git-scm.com)
+[<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/github/github-original.svg" alt="github" width="60" height="60"/>](https://github.com/)
+[<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/linux/linux-original.svg" alt="linux" width="60" height="60"/>](https://www.linux.org/)
+[<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/matlab/matlab-original.svg" alt="matlab" width="60" height="60"/>](https://www.mathworks.com/)
+[<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/numpy/numpy-original.svg" alt="numpy" width="60" height="60"/>](https://numpy.org/)
+[<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/pandas/pandas-original-wordmark.svg" alt="pandas" width="60" height="60"/>](https://pandas.pydata.org)
+[<img src="https://python-poetry.org/images/logo-origami.svg" alt="poetry" width="60" height="60"/>](https://python-poetry.org/)
+[<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/postgresql/postgresql-original-wordmark.svg" alt="postgresql" width="60" height="60"/>](https://www.postgresql.org/)
+[<img src="https://pre-commit.com/logo.svg" alt="precommit" width="60" height="60"/>](https://pre-commit.com/)
+[<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/python/python-original-wordmark.svg" alt="python" width="60" height="60"/>](https://www.python.org)
+[<img src="https://pypi.org/static/images/logo-large.9f732b5f.svg" alt="pypi" width="60" height="60"/>](https://pypi.org/)
+[<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/pytest/pytest-original-wordmark.svg" alt="pytest" width="60" height="60"/>](https://docs.pytest.org/)
+[<img src="https://brand-guidelines.readthedocs.org/_images/logo-wordmark-vertical-compact-dark.png" alt="readthedocs" width="60" height="60"/>](https://about.readthedocs.com)
+[<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/slack/slack-original.svg" alt="slack" width="60" height="60"/>](https://slack.com/)
+[<img src="https://space-packet-parser.readthedocs.io/en/latest/_static/logo-no-background.png" alt="space-packet-parser" width="60" height="60"/>](https://pypi.org/project/space-packet-parser/)
+[<img src="https://gitlab.inria.fr/uploads/-/system/project/avatar/42214/sphinxdoc.png" alt="sphinx" width="60" height="60"/>](https://www.sphinx-doc.org)
+[<img src="https://www2.jpl.nasa.gov/iae/highlights/yr-end99/SPICE/NaifLogoColor_shdw.jpg" alt="spice" width="60" height="60"/>](https://naif.jpl.nasa.gov/naif/toolkit.html)
+[<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/sqlalchemy/sqlalchemy-original-wordmark.svg" alt="sqlalchemy" width="60" height="60"/>](https://www.sqlalchemy.org/)
+[<img src="https://pydata.org/wp-content/uploads/2018/09/xarray-logo-square.png" alt="xarray" width="60" height="60"/>](https://docs.xarray.dev)
+
+
+### Useful Links
```

Comment:
I'd suggest moving this above the "languages, tools and libraries" section, as I think it will be more frequently accessed.